ush/gfs_bufr_netcdf.sh

```diff
@@ -82,7 +80,7 @@ do
    ic=0
    while [ $ic -lt 1000 ]
    do
-      if [ ! -f $COMIN/${RUN}.${cycle}.logf${hh2}.${fformat} ]
+      if [ ! -f $COMIN/${RUN}.${cycle}.logf${hh2}.txt ]
```

Comment:
This is a change (`txt` instead of `.nc`). Was the wrong filename being tested before?

Reply:
If you take a the COM. It should be the *logf*.txt. There is no such file called *logf*.nc